server/lease/lessor.go

```diff
@@ -394,6 +403,9 @@ func greaterOrEqual(first, second semver.Version) bool {
 // Renew renews an existing lease. If the given lease does not exist or
 // has expired, an error will be returned.
 func (le *lessor) Renew(id LeaseID) (int64, error) {
+	defer func(start time.Time) {
+		leaseRenewDurationSeconds.Observe(time.Since(start).Seconds())
```

Comment:
How this differs than grpc latency metrics for "leaseRenew" request?

Reply:
Ack, don't need this in favor of `grpc_server_handling_seconds_bucket{grpc_method="LeaseRenew"...}`. Will update PR.